plover/oslayer/osxkeyboardcontrol.py

```diff
@@ -222,6 +167,7 @@ def down_up(seq):
     36 : "Return", 124: "Right", 48: "Tab", 126: "Up",
 }
```

Comment:
I just noticed (well pylint did :P) that both BackSpace and Tab have the same keycode.